addons/l10n_vn/migrations/16.0.2.0.2/post-migration.py

```diff
@@ -0,0 +1,26 @@
+from odoo import api, SUPERUSER_ID
+from odoo.osv import expression
+
+FIXED_ACCOUNTS_TYPE = {
+    'expense_direct_cost': ['6271', '6272', '6273', '6274', '6277', '6278'],
+}
+
+
+def _fix_accounts_type(env):
+    for correct_account_type, accounts_code in FIXED_ACCOUNTS_TYPE.items():
```

Comment:
Are you planning to add more account types here ? the loop seems unnecessary for one  pair of  key-value.

Reply:
l10n_vn have a substantial number of accounts, and from time to time, Someone discover accounts that are misconfigured. I'm creating a general procedure to handle such issues in the future.